src/App.tsx

```diff
@@ -1,28 +1,55 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import { useForm, SubmitHandler } from 'react-hook-form';
+import { ApolloClient, InMemoryCache, ApolloProvider, useQuery, gql, useMutation } from '@apollo/client';
 
 type FormValues = {
   email: string;
   password: string;
 };
 
 function App() {
+  const [email, setEmail] = React.useState('');
+  const [password, setPassword] = React.useState('');
+  const [loginError, setLoginError] = React.useState('');
```

Comment:
aqui vc poderia instancia o erro como undefined

```suggestion
  const [loginError, setLoginError] = React.useState<string>();
```